core_memory_recall.py

```
@@ -0,0 +1,194 @@
+        # 初始化日志记录器
+        self.logger = logger
+
+    async def run(
+        self,
+        event: AstrMessageEvent,
+        limit: int,
+        query: str,
+    ) -> str:
+        self.logger.debug(f"{self.name} - LLM 调用: limit={limit}, query='{query}'")
+        if query is None or not str(query).strip():
+            return "参数错误：query 为必填且不能为空。请提供明确检索关键词后再调用 core_memory_recall。"
```

Ruff (0.15.12)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)